### DISCUSSION TITLE
What are the requirements of pybamm for computer configuration?

### DISCUSSION THREAD

**Discussion by RobinBluehub** [pybamm-team/PyBaMM#2156] [Q&A] [answered]
Hi, I would like to ask, does pybamm have a great impact on the running memory of the computer? My computer's 16g running memory is easily insufficient, which can easily cause the program to crash. In addition, will upgrading the processor reduce the solution time?

**Comment by brosaplanella** [comment, marked as answer]
16 GB should be enough, but it depends on what simulations you are running. If you run many cycles and store the entire solution you can easily run out of memory. [This notebook](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/simulating-long-experiments.ipynb) has some advice on how to run long experiments.
